pkg/cli/service.go

```diff
@@ -46,6 +57,196 @@ func NewSearchService() SearchService {
 	return searchService{}
 }
 
+func (service searchService) getImagesGQL(ctx context.Context, config searchConfig, username, password string,
+	imageName string,
+) (*imageListStructGQL, error) {
+	query := fmt.Sprintf(`{ImageList(repo: "%s") {`+`
+									RepoName Tag Digest ConfigDigest Size Layers {Size Digest}}
+							  }`,
+		imageName)
+	result := &imageListStructGQL{}
+
+	err := service.makeGraphQLQuery(ctx, config, username, password, query, result)
```

Comment:
This logic seems to repeat itself in multiple functions, we could move it to a separate function, that should improve with coverage.